### PR TITLE
Rework examples and tutorials regarding CC-NEWS

### PR DESCRIPTION
**Pull request by MaxDall**
This PR ...
* reworks example 3 in the README to display a more realistic CC-NEWS use case
* add a message about optimal process count when using the default process count
* rework tutorial about `CCNewsCrawler` a bit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,21 +107,33 @@ for article in crawler.crawl(max_articles=2):
     print(article)
 ```
 
-## Example 3: Crawl articles from CC-NEWS
+## Example 3: Crawl 1 Million articles
 
-If you're not familiar with CC-NEWS, check out their [paper](https://paperswithcode.com/dataset/cc-news).
+To crawl such a vast amount of data, Fundus relies on the `CommonCrawl` web archive, in particular the news crawl `CC-NEWS`.
+If you're not familiar with [`CommonCraw`](https://commoncrawl.org/) or [`CC-NEWS`](https://commoncrawl.org/blog/news-dataset-available) check out their websites.
```
Comment (alanakbik) [inline README.md:113]: small typo - should be: [`CommonCrawl`]

File: README.md [hunk 2]
```diff
@@ -107,21 +107,33 @@ for article in crawler.crawl(max_articles=2):
...
+Simply import our `CCNewsCrawler` and make sure to check out our [tutorial](docs/2_crawl_from_cc_news.md) beforehand.
 
 ````python
 from fundus import PublisherCollection, CCNewsCrawler
 
 # initialize the crawler for news publishers based in the US
-crawler = CCNewsCrawler(*PublisherCollection.us)
+crawler = CCNewsCrawler(*PublisherCollection)
```
Comment (alanakbik) [inline README.md:120]: Does not match the code comment about US publishers

File: README.md [hunk 3]
```diff
@@ -107,21 +107,33 @@ for article in crawler.crawl(max_articles=2):
...
 
 # crawl 2 articles and print
-for article in crawler.crawl(max_articles=2):
+for article in crawler.crawl(max_articles=1000000):
```
Comment (alanakbik) [inline README.md:123]: Does not match comment on 2 articles

File: README.md [hunk 4]
```diff
@@ -107,21 +107,33 @@ for article in crawler.crawl(max_articles=2):
...
   print(article)
 ````
 
+**Note**: By default, the crawler utilizes all available CPU cores on your system.
```
Comment (alanakbik) [inline README.md:127]: Also add a note on expected crawl time

**Review by alanakbik — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (MaxDall) [reply README.md:127]: Good idea 👍

File: README.md (hunk 3 above)
Comment (MaxDall) [reply README.md:123]: Fixed

File: README.md (hunk 2 above)
Comment (MaxDall) [reply README.md:120]: Fixed

File: README.md (hunk 1 above)
Comment (MaxDall) [reply README.md:113]: Thanks for catching 👍

**Review by addie9800 — APPROVED** [review]

**Review by alanakbik — APPROVED** [review]
Looks good!

**Pull request merged**